app/assets/javascripts/admin/addon/components/admin-config-areas/api-keys-new.gjs

```diff
@@ -31,11 +32,9 @@ export default class AdminConfigAreasApiKeysNew extends Component {
     { id: "single", name: i18n("admin.api.single_user") },
   ];
 
-  scopeModes = [
-    { id: "global", name: i18n("admin.api.scopes.global") },
-    { id: "read_only", name: i18n("admin.api.scopes.read_only") },
-    { id: "granular", name: i18n("admin.api.scopes.granular") },
-  ];
+  scopeModes = API_KEY_SCOPE_MODES.map((scopeMode) => {
+    return { id: scopeMode, name: i18n(`admin.api.scopes.${scopeMode}`) };
+  });
```

Comment:
🏌️ (not sure if your linting would allow it)/you can ignore if you do not like implicit returns:

```suggestion
  scopeModes = API_KEY_SCOPE_MODES.map((scopeMode) => ({ id: scopeMode, name: i18n(`admin.api.scopes.${scopeMode}`) })
  );
```

Reply:
I considered both. Decided on this mostly because adding more stuff into the object (forcing new lines) will produce a cleaner diff, and a little bit because there's some mild ambiguity with `{` as either object literal or function body. 🙂